crates/ty_ide/src/completion.rs

```diff
@@ -6374,7 +6385,14 @@ collabc<CURSOR>
         fn completion_test_builder(&self) -> CompletionTestBuilder {
             CompletionTestBuilder {
                 cursor_test: self.build(),
-                settings: CompletionSettings::default(),
+                settings: CompletionSettings {
+                    // The tests were originally written with auto-import
+                    // disabled, since it was disabled by default. But then
+                    // we enabled it by default. However, we kept the tests
+                    // as written with the assumption that auto-import was
+                    // disabled unless opted into. ---AG
+                    auto_import: false,
+                },
```

Comment:
Test builders that use different defaults as production is something that always makes me worried. It's so easy to write a test, think it passes when, in fact, it fails in production. I suggest we change the default but it's okay to do this in a separate PR